modules/serverless-channel-default.adoc

```diff
@@ -51,8 +57,7 @@ The channel controller then creates the backing channel instance based on the `s
 The `spec.channelTemplate` properties cannot be changed after creation, because they are set by the default channel mechanism rather than by the user.
 ====
 
-When this mechanism is used, two objects are created: a generic channel, and an `InMemoryChannel` channel.
-
-The generic channel acts as a proxy that copies its subscriptions to the `InMemoryChannel` channel, and sets its status to reflect the status of the backing `InMemoryChannel` channel.
+When this mechanism is used with the example above, two objects are created: a generic backing channel, and an `InMemoryChannel` channel.
+If you are using a different default channel implementation, for example, Apache Kafka, a generic backing channel and `KnativeKafka` channel are created.
```

Comment:
> for example, Apache Kafka, a generic backing channel and `KnativeKafka` channel are created.

Should be `KafkaChannel` , instead of `KnativeKafka`